src/SignalR/clients/ts/FunctionalTests/ts/Common.ts

```diff
@@ -120,12 +118,7 @@ export function getHttpClients(): HttpClient[] {
     if (typeof XMLHttpRequest !== "undefined") {
         httpClients.push(new XhrHttpClient(TestLogger.instance));
     }
-    if (typeof fetch !== "undefined") {
```

Comment:
Why stop checking if fetch is defined? Could this break scenarios that would work otherwise if XMLHttpRequest is defined, but fetch isn't?

Reply:
Nvm. I see this is test code.

Reply:
I think I'd still prefer to use `fetch` if it's defined rather than as a fallback. Almost all of our supported browsers have it (except IE 11).